docs/INSTALL-GUIDE-LINUX.md

```diff
@@ -0,0 +1,84 @@
+# Linux support
+
+This tutorial uses Ubuntu 20.04.
+
+Unless specified otherwise, all linux distributions are supported.
+
+## 1. install node.js and npm
+
+It is recommend to use [nvm (Node Version Manager)](https://github.com/nvm-sh/nvm#install--update-script),
+ use this link to install latest nvm version.
+
+For example:
+
+```Shell
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+```
+
+note: in the past we used to use `apt` for these instructions, but on ubuntu 20.04 the default
+ package provided is outdated, plus it creates broken dependencies when using curl to get latest
+ version from node.js's github.
+
+Then update your shell environment (for example with Ubuntu 20.04 in this example):
+
+```Shell
+source ~/.profile
+```
+
+Display available versions to install:
+
+```Shell
+nvm ls-remote
+```
+
+Install the node version you want, for example to install node 14.2.0.0:
+
+```Shell
+nvm install 14.2.0
+```
+
+see for details [the full tutorial's linux node install page](https://github.com/wonderingabout/gtp2ogs-tutorial/blob/master/docs/3A1-linux-download-install-nodejs.md).
+
+## 2. install gtp2ogs using npm
+
+Install gtp2ogs locally, for example at the root of you user account,
+ create a new folder `gtp2ogs-node`.
+
+```Shell
+mkdir ~/gtp2ogs-node
+```
+
+Then go inside this folder and install gtp2ogs with npm:
+
+```Shell
+cd ~/gtp2ogs-node
+npm install gtp2ogs
+```
+
+gtp2ogs will be installed in `~/gtp2ogs-node/node_modules/gtp2ogs`
+
+## 3. Recommended: Upgrade to devel branch
+
+This step is **highly recommended**.
```

Comment:
Note we're recommending people to switch to a beta version here.

Not per sé bad, it has a lot of new good features, but we should probably put a note about that here.

Also makes me think if we should add some branching strategy where users can use `master` and we work in `develop` branch or so. Only merging to master once we know it's stable for a bit.

Reply:
ps. this comment also count for the windows guide.

Reply:
yes, i'll explain that it is beta

can you explain the branching strategy in detail?

Reply:
is it how to do it ?

Reply:
is it how to do it?
https://stackoverflow.com/a/39732501

for example?
```Shell
npm install git://github.com/online-go/gtp2ogs.git#devel --save
```

Reply:
oh wow it works!!

![npm install devel](https://user-images.githubusercontent.com/38690718/81476116-48660900-9210-11ea-91a4-8f478be36bb8.png)

its not a git repo so we cant clone it, but at least we have latest devel natively

Reply:
found an even better way (now we clone from github and can use git in the repo), but we need to add node_modules and package-lock.json to git ignore

```Shell
git clone -b devel https://github.com/online-go/gtp2ogs && cd gtp2ogs && npm i
```

![npm install clone](https://user-images.githubusercontent.com/38690718/81476604-0ab6af80-9213-11ea-997e-cd1c66f95565.png)

from: https://stackoverflow.com/a/45847287